### ISSUE TITLE
Problem with imageEditButtons

### ISSUE THREAD

**Issue by svanetten1976** [froala/wysiwyg-editor#2235]
I am evaluating this WYSIWYG editor as part of a React CMS editor, in which I have many instances of the editor on the page. The problem I am running into is an intermittent one. SOMETIMES, when a new image is inserted into the editor and the imageEditButtons appear, clicking on one of the buttons (imageReplace, imageLink, etc.) fails to open the imageInsertButtons interface. Is there a workaround for this problem, or a way to programmatically open the imageInsertButtons interface? Very frustrating, very intermittent problem on an otherwise solid editor

**Comment by stefanneculai** [comment]
@svanetten1976 are you able to replicate this on our website? Otherwise, we would need a link where we can reproduce it. Thank you in advance.

**Comment by svanetten1976** [comment]
@stefanneculai thanks for replying so quickly. No, I have a very custom application in development inside our network. Can I ask you this -- because I am having a very intermittent problem with this feature, is there a way for me to manually hook into an event and programatically show the imageInsertButtons set when a user clicks on one of the imageEditButtons ('imageReplace', 'imageLink')? This is the only problem I'm having with this otherwise robust and fully featured editor.

Basically, can I do something like this? (forgive this rudimentary example):

```
'froalaEditor.imageEditButtons.imageReplace.click' : (e, editor) => {
   editor.imageInsertButtons.show()
}

```

**Comment by svanetten1976** [comment]
I figured this one out. Just FYI, using the editor.edit.on() and editor.edit.off() methods cause all kinds of problems with image editing. I have a toggling system for which I need the editor to have a read only mode, which Froala does not have a true option for. I figured out a workaround that involved disabling and enabling the toolbar and blocking the editor with an invisible div. The editor is great, but it would be a nice option to have a true readOnly mode
